### ISSUE TITLE
Unexpected token import error

### ISSUE THREAD

**Issue by iamdubx** [adeptoas/sweet-modal-vue#34]
This library gives me an error when using Nuxt/SSR.

```
[vue-router] Failed to resolve async component default: SyntaxError: Unexpected token import
SyntaxError: Unexpected token import
```

**Comment by ViniciusAugusto** [comment]
Hi @iamdubx, I'm having the same problem with component
![image](https://user-images.githubusercontent.com/3280420/29776961-bfd65636-8be0-11e7-90f5-72379e508164.png)

**Comment by frohikey** [comment]
I've ended up with importing vue file directly:  

``import SweetModal from 'sweet-modal-vue/src/components/SweetModal.vue'``

**Comment by ViniciusAugusto** [comment]
@frohikey It worked for me

thanks!!

**Comment by disitec** [comment]
There is a plan to fix this?

**Comment by bluefirex** [comment]
Yes there is

**Comment by disitec** [comment]
I did a PR that fix this.

**Comment by bluefirex** [comment]
Fixed in c05fb535b10cb39d2bfa17c218d198471a1e37da, released in [v2.0.0](https://github.com/adeptoas/sweet-modal-vue/releases/tag/v2.0.0)
